src/App.css

```diff
@@ -1,38 +1,84 @@
-.App {
-  text-align: center;
+body {
+  background-color: #ade146;
+  color: white;
+}
+
+main.home-main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+nav.top-bar {
+  display: flex;
+  justify-content: space-around;
+  align-items: baseline;
+  min-width: 95vw;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.nav-icon {
+  font-size: 28px;
+  margin: 0 5px 0 5px;
+  color: white;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+a.nav-link {
```

Comment:
- [ ] While you are free to pick a base color of your own it is a requirement to maintain layout: composition and space between elements of the original design. Kindly compare the layout for this element in the original design and make changes to your design to match it. 

![image](https://user-images.githubusercontent.com/64894927/137601554-5b22a62b-ecf1-4777-bb8d-9e59f871cee4.png)
![image](https://user-images.githubusercontent.com/64894927/137601572-d25e8e81-b881-46a1-8812-42d2c682e7c9.png)